packages/beta/src/api/documents/previewApi.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 Cognite AS
+
+import { BaseResourceAPI } from '@cognite/sdk-core';
+import { HttpResponseType } from '@cognite/sdk-core';
+
+import { DocumentId, DocumentPreviewTemporaryLink } from '../../types';
+import Any = jasmine.Any;
+
+export class PreviewAPI extends BaseResourceAPI<Any> {
+  public documentAsPdf = (id: DocumentId): Promise<ArrayBuffer> => {
+    return this.previewEndpoint<ArrayBuffer>(id, 'application/pdf');
+  };
+
+  public documentAsImage = (
+    id: DocumentId,
+    page: number
+  ): Promise<ArrayBuffer> => {
+    return this.previewEndpoint<ArrayBuffer>(id, 'image/png', page);
+  };
+
+  public temporaryLink = (
```

Comment:
is there any documentation planned? code snippets?

Reply:
yes, the documentation can be found here under /documents: https://docs.cognite.com/api/playground/#operation/getIcon

But did not plan to copy the documentation into the sdk in case the openapi documentation changes. Otherwise we will just bring up code snippets in our migration guide https://cognitedata.atlassian.net/wiki/spaces/search/pages/2923987180/Migration+guide+files+unstructured+documents

Reply:
Was thinking about adding the openapi link to each endpoint when we move to v1, if this is what you were thinking about?

Reply:
Yes, so you could do that later :)